services/learn-and-earn/src/notification.ts

```diff
@@ -0,0 +1,105 @@
+import { database, utils, interfaces } from '@impactmarket/core';
+import { fn, col, Op } from 'sequelize';
+import admin from 'firebase-admin';
+
+export async function availableCourses() {
+    const { learnAndEarnPrismicLevel, learnAndEarnUserLevel, appUser } = database.models;
+    const yesterday = new Date();
+    yesterday.setDate(yesterday.getDate() - 1);
+    yesterday.setHours(0, 0, 0, 0);
+
+    const newCourses = await learnAndEarnPrismicLevel.findAll({
+        where: {
```

Comment:
query only what is needed
```suggestion
        attributes: ['language'],
        where: {
```